onfido/helpers.py

```diff
@@ -50,22 +47,13 @@ def create_check(
     Returns a new Check object, and creates the child Report objects.
 
     """
-    if check_type != "standard":
-        raise ValueError(
-            f"Invalid check_type '{check_type}', currently only 'standard' "
-            "checks are supported."
-        )
-    if not isinstance(reports, (list, tuple)):
-        raise ValueError(
-            f"Invalid reports arg '{reports}', must be a list or tuple " "if supplied."
-        )
     data = {
-        "type": check_type,
-        "reports": [{"name": r} for r in reports],
+        "applicant_id": applicant.onfido_id,
+        "report_names": report_names,
     }
```

Comment:
This is really the only API change - when creating a Check we now POST to `/checks` and pass in the `applicant_id` as part of the data dict.